open-craft/xblock-dalite
# OC-1641 Change passport format and display different message when block is not set-up yet.

# Overview

This PR does two things: 
1. Switches LTI passport format so our passports are compatible with LTI passports irregardless of whether port is specified or not. 
2. Changes error message when block is not filled. 
# Testing instructions
1. Install xblock-dalite in your devstack
2. Run dalite (dalite has nice setup instructions in readme) 
3. Create a LTI passport for `dalite-xblock`. Note that format of the passport has changed, it is described in Readme of this repository. 
4. Add normal `LTI` module (described in good details in `dalite-ng` readme)
5. Add `dalite-xblock` in the same course, see that they can co-exist peacefully. 
6. Add a new xblock see that instructions for the instructors are now much clearer: 
   ![selection_002](https://cloud.githubusercontent.com/assets/112872/15515070/872a630e-21ec-11e6-8f0b-f9ad04ab84b5.png)
7. Run all tests, verify that coverage stays at 100%. 
8. Run make quality verify that no violations are set.
